### PR TITLE
Feat/define buttons and colors

### PR DESCRIPTION
**Pull request by aeschi**
- added hex colors to tailwind config
- added used font weights
- setup deafult font to IBM Plex Sans in dark gray
- added button components with options for disabling, adding icon and "destructive" option for tertiary button to appear in red
Screenshot 2024-03-07 at 15 12 53

### CODE REVIEW

**Comment by aeschi** [comment]
I left the buttons on the profile page for testing. Will delete them before merging.

File: public/fonts/IBMPlexSans-Bold.woff
Comment (raphael-arce) [inline public/fonts/IBMPlexSans-Bold.woff:1]: I think we don't need the `.woff` files, only the `.woff2` files

File: src/components/buttons/primary.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const PrimaryButton: React.FC<PrimaryButtonProps> = ({
+  label,
+  onClick,
+  disabled,
+}) => {
+  return (
+    <button
+      className={`text-gdk-white bg-gdk-blue hover:bg-gdk-light-blue disabled:bg-gdk-light-gray 
+      pointer-events-auto my-4 flex h-[51px] w-fit items-center justify-center rounded-[10px] px-8 font-semibold`}
```
Comment (raphael-arce) [inline src/components/buttons/primary.tsx:17]: i think we don't necessary need the pointer `events-auto-class` here (and in every button); I'd rather add it once to the parent container in which the buttons will be added to

File: src/components/buttons/tertiary.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  onClick: () => void;
+  disabled?: boolean;
+  destructive?: boolean;
+}
+
+const TertiaryButton: React.FC<TertiaryButtonProps> = ({
+  label,
+  onClick,
+  disabled,
+  destructive,
```
Comment (raphael-arce) [inline src/components/buttons/tertiary.tsx:14]: Right now, the destructive style is for the primary button defined in a different file, while for tertiary you can set it as a prop. I think we should choose only one way and be consistent on it. I think would prefer to have also a tertiary destructive in its own file. What do you think?

**Review by raphael-arce — CHANGES_REQUESTED** [review]
Looks pretty good already, thanks! I have commented a few suggestions.

File: src/components/buttons/primary.tsx (hunk 1 above)
Comment (aeschi) [reply src/components/buttons/primary.tsx:17]: yes, sounds good!

File: src/components/buttons/tertiary.tsx (hunk 2 above)
Comment (aeschi) [reply src/components/buttons/tertiary.tsx:14]: I agree, that would make it more consistent. Then we will have a "primary destructive" and "tertiary destructive" button for clear naming? @raphael-arce

Comment (raphael-arce) [reply src/components/buttons/tertiary.tsx:14]: @aeschi sounds great 👍

**Pull request merged**